naga/tests/in/glsl/quad_glsl.frag

```diff
@@ -6,6 +6,10 @@ layout(set = 0, binding = 1) uniform sampler u_sampler;
 #endif
 layout(location = 0) out vec4 o_color;
 
+struct FragmentOutput {
+  float x;
+};
+
```

Comment:
nitpick: I think it'd be better to have self-contained test(s) for the coverage specifically, rather than piggybacking in a way that's not clearly broken if the test gets changed for other reasons.

Reply:
I feel the same way - I just wasn't sure how people would feel. And for mysterious reasons I just happen to have a simple test case handy!